spec/vshasta_spec.sh

```diff
@@ -0,0 +1,17 @@
+Describe 'isgcp true'
+  Include sh/vshasta.sh
+  It "checks if a node is booted to a GCP image"
+    When call isgcp .
+      The output should eq ""
+      The status should be success
+    End
+End
+
+Describe 'isgcp false'
```

Comment:
Lets keep these tests a bit more consistent and test out failure cases as well, more like this we want to validate success cases as well as failures, this is what I could come up with:

```Describe "vshasta.sh"
  Include sh/vshasta.sh

  Context "isgcp valid cases"
   fixturedir=$(pwd)/vshasta.sh/etc
   fixturefile="${fixturedir}/google_system"

   Before 'setup'
   After 'teardown'

   setup() {
     install -dm755 ${fixturedir}
     install -m644 ${fixturefile}
   }
   teardown() {
     rm -fr ${fixturedir}
   }
   It 'only returns true for a file'
     When call isgcp ${fixturefile}
       The status should be success
     End
   End

  Context "isgcp failure cases"
    It 'fails on a non file'
      When call isgcp /dev/null
      The status should be failure
    End
    It 'fails on a dir'
      When call isgcp /usr
      The status should be failure
    End
    It 'fails on a nonexistent file'
      When call isgcp /does/not/exist
      The status should be failure
    End
  End
End

Reply:
fixed